apps/client/src/pages/builder/sidebars/left/sections/picture/section.tsx

```
@@ -22,20 +30,199 @@ import { PictureOptions } from "./options";
+        }
+
+        const croppedFile = new File([blob], selectedFile.name, {
+          type: selectedFile.type,
+          lastModified: Date.now(),
+        });
+
+        try {
+          const response = await uploadImage(croppedFile);
+          const url = response.data;
+          setValue("basics.picture.url", url);
+        } catch (error) {
```

ESLint

[error] Remove unused catch binding `error`.

(unicorn/prefer-optional-catch-binding)